generators/app/templates/infra/gcp/_settings.tf

```diff
@@ -0,0 +1,21 @@
+terraform {
+  backend "gcs" {
```

Comment:
This config has been put in a `backend.tf` file for aws, I think we should do the same

Reply:
As well as `provider.tf` and `versions.tf` files